setup.py

```diff
@@ -87,6 +88,9 @@ def __init__(self):
         self.library_paths += ["/usr/lib", "/lib"]
 
         for conf in ["/etc/ld.so.conf"] + glob.glob("/etc/ld.so.conf.d/*.conf"):
+            if not os.path.isfile(conf):
+                warnings.warn("File {} is broken.".format(conf))
```

Comment:
Can you be more specific in the warning. If I saw this during installation I had no clue what was going on.

Reply:
Yes.

Reply:
Ah ... I see what's going on. Why not be more specific? `os.path.islink(filename) and not os.path.exists(filename)`?

Reply:
I guess `exists` is fine here, because the symlink needs to be broken in order for this not to work.

Reply:
Ok :smile: Good point!